sites/postcss-preset-env/src/static/js/blog_color_mix_2023_03_27.js

```diff
@@ -4,15 +4,53 @@ import { tokenize } from '@csstools/css-tokenizer';
 
 const cylindrical = new Set(['hsl', 'hwb', 'lch', 'oklch']);
 const supportsP3AndP3Gamut = CSS.supports('(color: color(display-p3 0 0 0))') && window.matchMedia('(color-gamut: p3)').matches;
+const colorSpaceInput = document.getElementById('color-space');
+const interpolationMethodInput = document.getElementById('interpolation-method');
+const colorMixOutput = document.getElementById('output-color-mix');
+const colorMixOutputCSS = document.getElementById('output-color-mix-css');
+const colorMixPercentage = document.getElementById('color-mix-percentage');
+const colorInputA = document.getElementById('color-a');
+const colorOutputA = document.getElementById('output-color-a');
+const colorInputB = document.getElementById('color-b');
+const colorOutputB = document.getElementById('output-color-b');
+
+function debounce(fn, wait = 1) {
+	let timeout;
+
+	return function (...args) {
+		clearTimeout(timeout);
+		timeout = setTimeout(() => fn.call(this, ...args), wait);
+	};
+}
 
-function renderResult() {
-	const colorSpaceInput = document.getElementById('color-space');
-	if (!colorSpaceInput) {
-		return;
-	}
+function disableTransition() {
+	colorMixOutput.style.setProperty('transition', 'none');
+}
```

Comment:
I really like how the transitions work here and how they add an extra level of "mixing of colors" 🎉